dashboard/app/controllers/api/v1/pd/teacher_applications_controller.rb

```diff
@@ -30,7 +30,15 @@ def create
       application_json: application_json
     )
 
-    if @teacher_application.save
+    begin
+      @teacher_application.save
+    rescue
+      # Fail silently - this happens in the event of multiple submission
+      CDO.log.warn "Duplicate pd teacher application submission for user #{current_user.id} - ignoring the second one"
+      head :no_content
+    end
```

Comment:
I think you also need to return here. Otherwise it will continue below and render the errors on line 46 below (which presumably are empty so maybe it's ok?)